app/src/main/java/com/sun/music_66/view/adapter/ViewpagerFragmentAdapter.kt

```diff
@@ -0,0 +1,27 @@
+package com.sun.music_66.view.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.sun.music_66.view.GenreFragment
+import com.sun.music_66.view.LibraryFragment
+import com.sun.music_66.view.SettingFragment
+
+/**
+ * Created by nguyenxuanhoi on 2019-07-19.
+ * @author nguyen.xuan.hoi@sun-asterisk.com
+ */
+class ViewpagerFragmentAdapter(fragmentManager: FragmentManager) : FragmentPagerAdapter(fragmentManager) {
+
+    private var _listFragment = ArrayList<Fragment>()
```

Comment:
Tại sao anh không viết như này nhỉ? 
```suggestion
    private val fragments = listOf<Fragment>(
        GenreFragment.newInstance(),
        LibraryFragment.newInstance(),
        SettingFragment.newInstance()
    )
```